sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/aio/_configuration.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, endpoint: str, credential: "AsyncTokenCredential", **kwargs:
         self.credential = credential
         self.api_version = api_version
         self.credential_scopes = kwargs.pop("credential_scopes", ["https://cognitiveservices.azure.com/.default"])
-        kwargs.setdefault("sdk_moniker", "ai-language-conversations/{}".format(VERSION))
+        kwargs.setdefault("sdk_moniker", f"ai-language-conversations/{VERSION}")
```

Comment:
I'm just going to leave a comment on all the generated files so it's more obvious which ones shouldn't be edited.